apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/StylingCard.tsx

```diff
@@ -0,0 +1,215 @@
+"use client";
+
+import { Placement } from "@/app/(app)/environments/[environmentId]/settings/lookandfeel/EditPlacement";
```

Comment:
@pandeymangg please don't cross import from routes. We want to keep all routes independent from each other so we can safely delete a route or edit a component inside a route without needing to worry that it might affect some other place in the app.
If it's a commonly used UI component, move it to @formbricks/ui